doc/integrations.md

```diff
@@ -63,3 +67,36 @@ code_transform.indentation: 2
 # registered and hope for the best.
 autoload.deregister: false
 ```
+
+The downside to this option is that it requires access to the DB from your current environment which may be tricky if you are running Drupal inside a VM.
+
+### Option 2: Add the modules into the Composer autoloader
+
+This option requires merging in any modules (Drupal, contrib, custom) into the Composer autoloader via a discovery mechanism offered by the [Drupal Autoloader](https://github.com/fenetikm/autoload-drupal) composer plugin.
+
+Full details are available in the README for this plugin, however, the short version is that you will need to put the following into your `composer.json` file:
+
+```json
+{
+    "require": {
+        "fenetikm/autoload-drupal": "dev-master"
```

Comment:
Would be nice to suggest to tag this, as people will actually be adding it their projects and it `dev-master` cold break...